### PR TITLE
Remove freenode mentions in remaining files

### LINKED ISSUE #2702: Remove freenode mentions in remaining files
We now moved to libra chat as rizin did here https://github.com/rizinorg/website/commit/24309365e91073e1e1c3c02acf1c01ec576b3995.
We should update the links in the application itself as well as on the website: https://github.com/rizinorg/cutter.re

```
$ git grep 'freenode' | awk -F: '{print $1}' | uniq
.github/ISSUE_TEMPLATE/config.yml
README.md
docs/source/index.rst
src/dialogs/WelcomeDialog.ui
```

### PR DESCRIPTION
**Pull request by rbrishabh**
Fixes #2702

Hi @xarkes! How does this look?
let me know if this needs any changes!
Thanks

### CODE REVIEW

**Comment by 2702** [comment]
Hi, why am I tagged here?

**Comment by rbrishabh** [comment]
My bad! It was a mistake

**Review by XVilka — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -6,5 +6,5 @@ contact_links:
     about: Please ask questions about Cutter here or one of the other community channels, not in the issue tracker.
 
   - name: Questions IRC
-    url: https://webchat.freenode.net/#cutter
-    about: "#cutter on irc.freenode.net"
\ No newline at end of file
+    url: https://kiwiirc.com/nextclient/#irc:/irc.libera.chat/#cutter
```
Comment (Surendrajat) [inline .github/ISSUE_TEMPLATE/config.yml:9]: I guess we need it in format `irc://...`, right?
```suggestion
    url: https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#cutter
```

**Review by Surendrajat — CHANGES_REQUESTED** [review]
BTW libera.chat has their own webchat(Kiwi IRC) hosted on https://web.libera.chat which can be used as the link.

**Comment by XVilka** [comment]
@Surendrajat interesting, that wasn't the case when I did the change for the rizin.re site. We should probably use it instead. And update the links everywhere in rizinorg.

**Comment by rbrishabh** [comment]
@XVilka If you want, I can do that on the other repos! you can just create an issue and tag me! :)

**Comment by rbrishabh** [comment]
wait, so the changes reqd are that we need to change:
`https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#cutter` with `https://web.libera.chat/#cutter`?

**Comment by Surendrajat** [comment]
@rbrishabh yes.

**Comment by ITAYC0HEN** [comment]
> wait, so the changes reqd are that we need to change:
> `https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#cutter` with `https://web.libera.chat/#cutter`?

Exactly! Indeed few weeks ago it wasn't the case. It should be updated for:
- Rizin website (https://github.com/rizinorg/website) - Notice that there are multiple channels (one for users and one for devs)
- Rizin README and other places if needed (https://github.com/rizinorg/rizin)
- Cutter Source (https://github.com/rizinorg/cutter)
- Cutter Website (https://github.com/rizinorg/cutter.re)

**Comment by rbrishabh** [comment]
Sure! I can do this later today!

**Comment by ITAYC0HEN** [comment]
That's great! Let us know if you need any help. You can start from this PR on Cutter and then move forward

**Comment by xarkes** [comment]
Hey @rbrishabh, thanks for the PR, changes look great as said by the others :)
If you can specify the official link as mentioned before instead of kiwiirc it would be perfect!

**Comment by rbrishabh** [comment]
Sure @xarkes! I had a slightly busy weekend but I plan to get to this first thing on Monday!
sorry for the slight delay :)

**Comment by xarkes** [comment]
No problem, and no worries! :)

**Comment by rbrishabh** [comment]
Hey guys! this is ready for another review!
I'll make the same changes on other repositories!
@xarkes @Surendrajat @XVilka 

Thank you

**Review by Surendrajat — APPROVED** [review]
Looks good. Thanks!

**Review by xarkes — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**